.github/actions/spelling/patterns.txt

```diff
@@ -91,6 +91,7 @@ value="(?:[0-9a-f]{1,2} )*"
 
 # Ignore any text between inline back-ticks
 `(.*?)`
+```(.*?)```
```

Comment:
This won't do what you're expecting... check-spelling doesn't currently support multi-line concepts, and this pattern would be covered by the inline back-ticks pattern.

I'm still thinking about the steps necessary to handle multiline patterns (the current file really won't support it, so I basically need to imagine a new file and a new syntax for the file, or something) -- https://github.com/check-spelling/check-spelling/wiki/Feature%3A-Block-Ignore

Reply:
Understood, thank you for the explanation. I have indeed removed it from the PR I opened cause I noticed it didn't work.